Resources/Prototypes/Entities/Objects/Weapons/Melee/stunprod.yml

```diff
@@ -0,0 +1,49 @@
+- type: entity
+  name: stun prod
+  parent: BaseItem
+  id: Stunprod
+  description: A stun prod for illegal incapacitation.
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Melee/stunprod.rsi
+    netsync: false
```

Comment:
Unneeded

Reply:
what exactly? there will be no sprite if I remove it